tests/src/Kernel/Update405Test.php

```diff
@@ -21,16 +21,34 @@ class Update405Test extends KernelTestBase {
    * @covers ::enableAutosaveForm
    */
   public function testEnableAutosaveForm() {
-    /** @var \Drupal\Core\Extension\ModuleHandlerInterface $module_handler */
-    $module_handler = $this->container->get('module_handler');
-    $this->assertFalse($module_handler->moduleExists('autosave_form'));
-    $this->assertFalse($module_handler->moduleExists('conflict'));
+
+    $moduleHandler = $this->container->get('module_handler');
+
+    $this->assertFalse($moduleHandler->moduleExists('autosave_form'));
+    $this->assertFalse($moduleHandler->moduleExists('conflict'));
 
     Update405::create($this->container)->enableAutosaveForm();
 
-    $module_handler = $this->container->get('module_handler');
-    $this->assertTrue($module_handler->moduleExists('autosave_form'));
-    $this->assertTrue($module_handler->moduleExists('conflict'));
+    $moduleHandler = $this->container->get('module_handler');
+
+    $this->assertTrue($moduleHandler->moduleExists('autosave_form'));
+    $this->assertTrue($moduleHandler->moduleExists('conflict'));
+  }
+
+  /**
+   * @covers ::enableRedirect
+   */
+  public function testEnableRedirect() {
```

Comment:
So...this actually doesn't need to be in its own method. I would suggest we rename testEnableAutosaveForm() to testUpdate(), and have it do something like this:

```php
$moduleHandler = $this->container->get(...);

$this->assertFalse($moduleHandler->moduleExists('autosave_form'));
// Same thing for Conflict and Redirect

$update = Update405::create($this->container);
$update->enableAutosaveForm();
$update->enableRedirect();

$moduleHandler = ...
$this->assertTrue($moduleHandler->moduleExists('autosave_form'));
// ...etc.
```
This allows us to test everything we need to, and keep things a bit simpler and less verbose. :)

Reply:
makes sense!